src/views/TabView/TabBarBottom.js

```diff
@@ -57,6 +58,13 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
 
   props: Props;
 
+  _onTabPress = (scene: TabScene) => {
+    const { getTabPressCallback, jumpToIndex } = this.props
+    const onTabPress = getTabPressCallback(scene);
+    onTabPress(scene);
```

Comment:
Is there a better way to block navigation to a tab than

`
if !!onTabPress {
  jumpToIndex
}
`